Cargo.lock

```diff
@@ -1872,9 +1872,9 @@ dependencies = [
 
 [[package]]
 name = "once_cell"
-version = "1.12.0"
+version = "1.13.0"
```

Comment:
I guess the changes to `Cargo.lock` should be removed too.

Reply:
Hm, interesting that this didn't go away....